tests/mainTest.py::footer::testSocials
--------------------------------------------------------------------
Last Page: file:///C:/Users/Nikole/Documents/GitHub/Bengans-Biluthyrning/index.html
  Browser: chrome (105.0.5195.102)
   Driver: chromedriver (105.0.5195.52)
Timestamp: 1663149533  (Unix Timestamp)
     Date: Wednesday, 14 September 2022
     Time: 11:58:53 AM  (W. Europe Daylight Time, UTC+01:00)
--------------------------------------------------------------------
Traceback: File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.2032.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 591, in run
    self._callTestMethod(testMethod)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.2032.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 549, in _callTestMethod
    method()
  File "C:\Users\Nikole\Documents\GitHub\Bengans-Biluthyrning\tests\mainTest.py", line 47, in testSocials
    self.click(f"[src=\"{socialMediaPaths[i]}\"]")
  File "C:\Users\Nikole\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\seleniumbase\fixtures\base_case.py", line 281, in click
    element = page_actions.wait_for_element_visible(
  File "C:\Users\Nikole\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\seleniumbase\fixtures\page_actions.py", line 423, in wait_for_element_visible
    timeout_exception(NoSuchElementException, message)
  File "C:\Users\Nikole\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\seleniumbase\fixtures\page_actions.py", line 192, in timeout_exception
    raise exc(msg)
Exception: Message: 
 Element {[src="src/images/svg/facebookIcon.svg"]} was not present after 7 seconds!
